roles/openshift_grafana/tasks/main.yml

```diff
@@ -6,35 +6,24 @@
 
 - name: Configure Grafana Permissions
   include_tasks: tasks/gf-permissions.yml
-  when: gf_oauth | default(false) | bool == true
-
-# TODO: we should grab this yaml file from openshift/origin
-- name: Templatize grafana yaml
-  template: src=grafana-ocp.yaml dest=/tmp/grafana-ocp.yaml
-  register:
-    cl_file: /tmp/grafana-ocp.yaml
-  when: gf_oauth | default(false) | bool == false
-
-# TODO: we should grab this yaml file from openshift/origin
-- name: Templatize grafana yaml
-  template: src=grafana-ocp-oauth.yaml dest=/tmp/grafana-ocp-oauth.yaml
-  register:
-    cl_file: /tmp/grafana-ocp-oauth.yaml
-  when: gf_oauth | default(false) | bool == true
+
+- name: Pull grafana yaml from openshift/origin
+  get_url:
+    url: https://raw.githubusercontent.com/openshift/origin/master/examples/grafana/grafana.yaml
```

Comment:
Do we need to hard code a URL to a **different** repository in here? What if the other repository deletes or moves this file? Use a URL variable here `{{ graphana_base_url }}/graphana.yaml` and set the `graphana_base_url` in the `.../defaults/main.yml` file so users can override or change this URL.